src/resources/uart_posix.cc

```diff
@@ -103,6 +117,9 @@ static int int_to_linux_baud_rate(int baud_rate, speed_t* speed) {
     case 3500000: *speed = B3500000; return 0;
     case 4000000: *speed = B4000000; return 0;
     default: return -1;
+#elifdef TOIT_DARWIN
+    default: *speed = baud_rate; return 0;
```

Comment:
In theory there could be a clash between the baud rate and the Bxxx defines?  Perhaps should set a flag indicating that we should skip tcgetattr/tcsetattr and go straight to IOSSIOSPEED.